doc/tutorial/faq_tutorial.txt

```diff
@@ -64,7 +64,7 @@ Defining the theano function
 
 .. code-block:: python
 
-   f = theano.function(..., updates=updates)
```

Comment:
What is this new thing about lookup tables for updates?  Since when is that in?  Why did we not mention that in the release notes?